test/spec/airbyte-local_spec.sh

```diff
@@ -246,4 +246,39 @@ Describe 'building destination catalog'
 
         The output should include 'Using destination configured catalog: {"streams":[{"stream":{"name":"dummy_prefix__faros_feed","json_schema":{}},"sync_mode":"incremental","destination_sync_mode":"append"}]}'
     End
+
+    It 'creates stream prefix when source and destination are Faros connectors'
+        When run source ../airbyte-local.sh \
+                --src 'farosai/airbyte-dummy-source-image' \
+                --dst 'farosai/airbyte-faros-destination' \
+                --debug
+
+        The output should include 'Using destination configured catalog: {"streams":[{"stream":{"name":"mydummysourcesrc__dummy_source__faros_feed","json_schema":{}},"sync_mode":"incremental","destination_sync_mode":"append"}]}'
+    End
+    It 'creates stream prefix including connection_name when source and destination are Faros connectors'
+        When run source ../airbyte-local.sh \
+                --src 'farosai/airbyte-dummy-source-image' \
+                --dst 'farosai/airbyte-faros-destination' \
+                --connection-name 'connectionXYZ' \
+                --debug
+
+        The output should include 'Using destination configured catalog: {"streams":[{"stream":{"name":"connectionXYZ__dummy_source__faros_feed","json_schema":{}},"sync_mode":"incremental","destination_sync_mode":"append"}]}'
+    End
+    It 'fails if missing stream prefix when using Faros destination and non-Faros source'
+        When run source ../airbyte-local.sh \
+                --src 'airbytehq/airbyte-dummy-source-image' \
+                --dst 'farosai/airbyte-faros-destination' \
+                --debug
+
+        The output should include "farosai/airbyte-faros-destination requires a destination stream prefix. Specify this by adding '--dst-stream-prefix <value>'"
```

Comment:
The output arg name does not match

Reply:
Wdym? Not sure I understand.

When using Faros destination and non-Faros source, a destination prefix (specified via `--dst-stream-prefix`) is required. This is what this test is about